client-java/pom.xml

```diff
@@ -69,12 +71,6 @@
                 <artifactId>slf4j-api</artifactId>
                 <version>${slf4j.version}</version>
             </dependency>
-            <dependency>
-                <groupId>org.junit.jupiter</groupId>
-                <artifactId>junit-jupiter</artifactId>
-                <version>${junit.jupiter.version}</version>
-                <scope>test</scope>
-            </dependency>
             <dependency>
                 <groupId>org.junit.jupiter</groupId>
                 <artifactId>junit-jupiter-api</artifactId>
```

Comment:
I think that we only need to declare a dependency for artifactId   junit-jupiter-engine.  The others do not need to be included in the file as they are pulled by declaring this dependency.  Could be worth a check

Reply:
Removed the jupiter-params dependency as this did not have an affect on the build process. Removing any of the the others either failed the build or provided a warning message that the tests could not be run.